modules/Makefile.kubernetes

```diff
@@ -85,18 +87,47 @@ kubernetes\:replace-controller:
 kubernetes\:replace:
 	@$(SELF) kubernetes:replace-service kubernetes:replace-controller
 
+# (private) Create a new deployment
+kubernetes\:create-deployment:
```

Comment:
When would we use this vs `apply-deployment`?

The k8s docs are unclear. The example they provide shows a create for the initial deployment, followed up apply for subsequent ones.

http://kubernetes.io/docs/user-guide/deployments/#creating-a-deployment